barretenberg/cpp/src/barretenberg/honk/transcript/transcript.hpp

```diff
@@ -186,23 +229,11 @@ template <typename FF> class BaseTranscript {
         return challenges;
     }
 
-    FF get_challenge(const std::string& label) { return get_challenges(label)[0]; }
-
-    [[nodiscard]] TranscriptManifest get_manifest() const { return manifest; };
-
-    void print() { manifest.print(); }
-};
-
-template <typename FF> class ProverTranscript : public BaseTranscript<FF> {
```

Comment:
deleting ProverTranscript and VerifierTranscript and merging it with BaseTranscript so that I do not have to define Prover/Verifier Transcripts that inherit from each flavor Transcript class.